packages/components/tests/dummy/app/templates/components/button.hbs

```diff
@@ -60,12 +64,14 @@
     </dd>
     <dt>isIconOnly <code>boolean</code></dt>
     <dd>
+      <p>Default: <span class="default">false</span></p>
```

Comment:
I added 2 "default" tags here for clarity but let me know if you want me to undo these changes for this PR.